rust/macros/helpers.rs

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0
```

Comment:
I believe the right tag is Co-developed-by and a sign-off is required from everyone who is listed as Co-developed-by.

Reply:
I guess I can just copy the signoff from the original commits then? It's just that one simple macro which is already in the `rust` branch (I just followed the history to make sure I got everyone who touched it).

Reply:
I believe so based on reading https://docs.kernel.org/process/submitting-patches.html#when-to-use-acked-by-cc-and-co-developed-by

Reply:
Done!